packages/terra-clinical-application-site/src/index.scss

```diff
@@ -16,6 +16,7 @@ body {
   height: 100%;
 }
 
-.terraClinical-ItemView-accessory {
+[class*="ItemView__accessory"],
+[class*="ItemView__accessory"] > * {
```

Comment:
The Item View styling gives accessory items a max-width of 60px. The accessory provided in the demo exceeds this maximum width, which is why the styling is being overridden.